tests/baselines/reference/quickInfoJsDocTags13.baseline

```diff
@@ -25,12 +25,14 @@
 // | ----------------------------------------------------------------------
 // | f(**a: number**): void
 // | First overload
+// | @overload 
```

Comment:
often "redundant" `@param` (and more are rendered) - is this any different? I think it's fine to leave it as-is right now and redundant tags could be tackled separately if that's a desired feature.